audit/audit.go

```diff
@@ -15,6 +15,10 @@ const (
 	// OpSetPerm represents the setting of ACLs on an entity.
 	// Required fields: Entity, ACL
 	OpSetPerm Operation = "set-perm"
+
+	// OpSetPromulgation represents the promulgation on an entity.
+	// Required fields: Entity, Promulgation
+	OpSetPromulgation Operation = "set-promulgation"
```

Comment:
"set-promulgated" (and OpSetPromulgated) would seem to fit better with the existing names in the code and API.